docs/toc.yml

```
@@ -45,6 +47,11 @@
       href: reference/messaging.md
     - name: AI
       href: reference/ai.md
+    - name: Flow
+      href: reference/flow.md
+      items:
+      - name: Bindings and canonical IDs
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)